libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SFKeychainItemWrapper.m

```diff
@@ -36,8 +36,6 @@ @interface SFKeychainItemWrapper ()
 
 @implementation SFKeychainItemWrapper
 
-@synthesize passcodeLength = _passcodeLength;
-
 // NSException constants
```

Comment:
Consideration for your next PR: Can we get a newer version of this file that's ARC-enabled? Then we can get rid of the special compiler flag for this file (`-fno-objc-arc`). The following steps will show you where that is:
- `SalesforceSDKCore`
- `Build Phases`
- `Compile Sources`
- Next to `SFKeychainItemWrapper.m`, you'll see the flag.